src/map.cpp

```diff
@@ -2951,11 +2971,15 @@ std::pair<bool, bool> map::bash_ter_furn( const tripoint &p, const int str,
                 }
                 sound_volume = 8;
                 sound = _("rrrrip!");
+                soundfxid = _("smash_success");
+                soundfxvariant = _("smash_cloth");
                 smashed_something = true;
                 success = true;
             } else {
                 sound_volume = 8;
                 sound = _("slap!");
+                soundfxid = _("smash_fail");
+                soundfxvariant = _("hit_cloth");
```

Comment:
Aren't those strings here supposed to be ids, they should probably not be translated (the `_()` translates the string).